src/traces/sankey/constants.js

```diff
@@ -15,8 +15,8 @@ module.exports = {
     sankeyIterations: 50,
     forceIterations: 5,
     forceTicksPerFrame: 10,
-    duration: 500,
-    ease: 'cubic-in-out',
+    duration: 350,
+    ease: 'quart-in-out',
```

Comment:
Why?

Reply:
`duration: 350` for it to complete before saving to image
`ease: 'quart-in-out'` just makes it much easier to follow the trajectories of nodes on relayout/grouping. I guess, we should have a discussion about animations.

Reply:
Ha I see. Voting :-1: on animations.

Reply:
About animations: in this [Codepen](https://codepen.io/antoinerg/pen/MLxjmB), toggling between `Ungroup [[]]` and `Group [[[0,2]]]`  shows where the animations can shine. It really helps to see what nodes are being merged/unmerged.

I'm also on the fence about enabling them because of the timeout I had to add to `src/snapshot/helpers.js`. On the other hand, I think the animations do add value and can look great.

Mabye @alexcjohnson has an opinion on this one?

BTW, we can simply comment out the couple `.style('opacity', 0)` calls to get rid of the new animations added in this PR so no additional work will be required either way.

Reply:
Animation for sankey trace will be useful, no doubt. But I don't think they should be triggered from plain `restyle` calls. Our other traces don't do that. I think we should try to make sankey animations follow how `scatter` animations behave.

@antoinerg you can try playing around with `transitionOpts` and `makeOnCompleteCallback` as  in [`Scatter.plot`](https://github.com/plotly/plotly.js/blob/7da95e1af39b4b1f98abc2a43deb6b117435931d/src/traces/scatter/plot.js#L25). You'll need those to get animations to end properly (i.e. return a Promise) and have access to the transition options.

Reply:
@etpinard If you interact with the menu in mock `sankey_energy` you will see animations on `restyle`. If I :hocho: animations, can those ones also be removed? Would you consider that a breaking change?

Reply:
Arrg. We'll have to keep the existing animations. Otherwise yes, that would be a breaking change.

Reply:
Resolved!